src/utils/CopilotAPI.ts

```diff
@@ -96,6 +96,10 @@ export class CopilotAPI {
     return CompaniesResponseSchema.parse(await this.copilot.listCompanies(args))
   }
 
+  async getCompanyClients(companyId: string): Promise<ClientResponse[]> {
+    return (await this.getClients({ limit: 10000, companyId })).data || []
```

Comment:
Yes, the limit is mandatory because we want to bulk create notifications for all company clients. Chunking will be later handled by the bulkNotifications service method